src/dune_rules/ocamldep.mli

```diff
@@ -3,8 +3,24 @@
 open! Dune_engine
 open Import
 
+module Modules_data : sig
```

Comment:
Plural in `Modules` seems a bit strange to me, I'd probably just call this `Module_data`.

Reply:
I used the plural form because it's about a group of modules. To be fair, the name I didn't really know how to call this.